pkgs/development/python-modules/wgnlpy/default.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  cryptography,
+  pyroute2,
+}:
+
+buildPythonPackage rec {
+  pname = "wgnlpy";
+  version = "0.1.5";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "ArgosyLabs";
+    repo = "wgnlpy";
+    rev = "v${version}";
+    hash = "sha256-5XAfBiKx4SqouA57PxmaCb0ea7mT2VeUI1tgnQE/ZwQ=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [
+    cryptography
+    pyroute2
+  ];
+
+  pythonImportsCheck = [
+    "wgnlpy"
+    "wgnlpy.nlas"
+  ];
+
```

Comment:
```suggestion

  doCheck = false; # no tests

```

Reply:
Not recommended if `checkPhase` doesn't bring much overhead, as this is more likely to be ignored even after an update with tests added.